### PR TITLE
fix(BadgeList):#MBA-41:fix css more and align items

### PR DESCRIPTION
**Pull request by Rseuret**
## Describe your changes
CSS adjustements
## Issue ticket number and link
https://jira.support-ent.fr/browse/MBA-41
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [ ] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/sass/global/components/_badge.scss [hunk 1]
```diff
@@ -9,7 +9,11 @@
       opacity: 0.5;
     }
 
+    &-label{
+      margin-left:0.2%;
```
Comment (JinDouang) [inline src/main/resources/public/sass/global/components/_badge.scss:13]: que représente ces valeurs 0.2/0.3% ?
On peut pas utiliser les multiples de 8 ? genre 8px 16 px etc...

File: src/main/resources/public/sass/global/components/_card.scss [hunk 2]
```diff
@@ -99,7 +99,12 @@
             @include paddingX(28px);
             @include paddingY(6px);
             white-space: nowrap;
+            justify-content: flex-start;
 
+            & .lock{
+              padding-left: 2.5px;
```
Comment (JinDouang) [inline src/main/resources/public/sass/global/components/_card.scss:105]: pourquoi 2.5px ? Pas possible d'avoir un multiple de 8 ?

**Review by JinDouang — CHANGES_REQUESTED** [review]

File: src/main/resources/public/ts/directives/badge-list/badge-list.html [hunk 3]
```diff
@@ -1,9 +1,12 @@
 <div class="minibadge-badge minibadge-badge-list minibadge-container flex-row f-column"
      ng-class="{ disabled: !vm.badges || !vm.badges.length }">
     <h3 ng-click="vm.clickList()">
-        <i class="play" ng-class="{ opened: vm.isOpenedList && vm.badges && !!vm.badges.length }"></i>&nbsp;<i18n>
-        [[vm.lang.translate(vm.label)]]
-    </i18n>
+        <i class="play" ng-class="{ opened: vm.isOpenedList && vm.badges && !!vm.badges.length }"></i>&nbsp;
+        <span class="minibadge-badge-list-label">
+            <i18n>
```
Comment (qpic-cgi) [inline src/main/resources/public/ts/directives/badge-list/badge-list.html:6]: En réalité, la balise i18n n'est pas nécessaire du coup.

File: src/main/resources/public/sass/global/_mixins.scss [hunk 4]
```diff
@@ -31,6 +32,10 @@ $minibadge-shadow-light: rgba(214, 214, 214, 0.3);
   padding-top: $size;
 }
 
+@mixin paddingXY($sizeX, $sizeY) {
+  padding: $sizeY $sizeX;
+}
```
Comment (qpic-cgi) [inline src/main/resources/public/sass/global/_mixins.scss:37]: pas idiot ! 👍🏻

**Review by sjollois — APPROVED** [review]

**Pull request merged**